sr_fingertip_visualization/src/sr_fingertip_visualization/tactile_point_generic.py

```diff
@@ -39,7 +38,7 @@ def __init__(self, index=None, parent=None):
         self._center = QPoint(self.frameSize().width()/2, self._radius + self._title_height)
         self.setMinimumSize(self._MIN_SIZE_X, 2*self._radius)
 
-    def paintEvent(self, event):
+    def paintEvent(self, _event):  # pylint: disable=C0103
```

Comment:
If you added an _ at the beginning of event because it is not used, you should rather add it at the end, like event_, or just keep it to event? Starting with _ means that it is something private or protected

Reply:
Ah cool, I will do that :)